CONTRIBUTING.md

```diff
@@ -22,10 +22,17 @@ $ git rebase upstream/develop
 ```
 * Please submit separate pull requests for different features; i.e. try not to shove multiple, unrelated changes into one pull request.
 * Please make sure the pull request only contains changes that you've made intentionally; if you didn't mean to touch a file, you should probably not include it in your commit. Here are some files that like to have spurious changes in them:
-  - `Podfile.lock`: This file may change if you have a different CocoaPods version installed than the project maintainers. We suggest updating to the ~~latest release~~ whatever `sudo gem install cocoapods --pre` gives you, at least until CocoaPods 1.6 is released.
   - `project.pbxproj`: This file may change if you sign the project with a different developer account. Changes due to adding or removing files are OK, generally.
   - `xib` files: Please discard changes to an `xib` file if you didn't change anything in it.
 
+### Localizations
+
+We use [Crowdin](https://crowdin.com/project/iina) for localization.
+When there is a UI text change, only add/update the strings both in `Base.lproj` and `en.lproj`.
+Please do not contain any localization changes in other languages in the pull request.
```

Comment:
Use "include" instead of "contain", "for" instead of "in":
Please do not include any localization changes for other lnaguages in the pull request.